src/ReportManager/DetailsSection/index.js

```diff
@@ -234,6 +235,12 @@ const DetailsSection = ({
       <button ref={submitFormButtonRef} type="submit" />
     </Form>}
 
+    {!!formSchema && isEFBSchema &&
+      <EFBForm schema={formSchema}
```

Comment:
I want to know, what do you think of the props that we pass to that other Form component? Like `onFormSubmit`, `onFormError`, `formValidator`... I know we don't need to follow that old way to interact with the form, but we need somehow to make it work similar @gaboDev